docs/sources/explore/get-started-with-explore.md

```diff
@@ -0,0 +1,210 @@
+
+Based on specific data source, certain query editors allow you to select the label or labels to add to your query. Labels are fields that consist of key/value pairs representing information in the data. Some data sources allow for selecting fields.
+
+1. Click **Run query** in the upper right to run your query.
+
+## Content outline
+
+The content outline is a side navigation bar that keeps track of the queries and visualizations you created in Explore. It allows you to navigate between them quickly.
+
+The content outline works in a split view, with a separate outline generated for each pane.
+
+To open the content outline:
+
+1. Click the Outline button in the top left corner of the Explore screen.
+
+You can then click on any panel icon in the content outline to navigate to that panel.
+
+## Split and compare
+
+The split view enables easy side-by-side comparison of visualizations or simultaneous viewing of related data on a single page.
+
+To open the split view:
+
+1. Click the split button to duplicate the current query and split the page into two side-by-side queries.
+1. Run and re-run queries as often as needed.
+
+You can select a different data source, or different metrics and label filters for the new query, allowing you to compare the same query across two different servers or compare the staging environment with the production environment.
+
+{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-explore-split-10.1.png" max-width= "950px" caption="Screenshot of Explore screen split" >}}
+
+You can also link the time pickers for both panels by clicking on one of the time-sync buttons attached to the time pickers. When linked, changing the time in one panel automatically updates the other, keeping the start and end times synchronized. This ensures that both split panels display data for the same time interval.
+
+Click **Close** to quit split view.
+
+## Time picker
+
+Use the time picker to select a time range for your query. The default is **last hour**. You can select a different option from the dropdown or use an absolute time range. You can also change the timezone associated with the query, or use a fiscal year.
+
+1. Click **Change time settings** to change the timezone or apply a fiscal year.
+
+Refer to [Set dashboard time range](https://grafana.com/docs/docs/grafana/<GRAFANA_VERSION>/dashboards/use-dashboards/#set-dashboard-time-range) for more information on absolute and relative time ranges. You can also [control the time range using a URL](https://grafana.com/docs/docs/grafana/<GRAFANA_VERSION>/dashboards/use-dashboards/#control-the-time-range-using-a-url).
```

Comment:
Both of these links 404 for me in the local build

Reply:
will redo links